.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  name: Lint
+
+  runs-on: ubuntu-latest
+
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: recursive
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: lint
+      run: |
+        go get -u honnef.co/go/tools/cmd/staticcheck@latest &&
+        $HOME/go/bin/staticcheck
+
+jobs:
+  name: Test
+
+  runs-on: ubuntu-latest
+
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: recursive
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: test
+      run: |
+        go generate &&
```

Comment:
I don't know if go-bindata exists in the default build container, in goreleaser, it's installed as pre-req: https://github.com/stripe/stripe-mock/pull/266/files#diff-1c7213c2f02fce0c1d653f1fe32831b700931145e3d5b7245c0df732c492aba8R6